packages/office-ui-fabric-react/src/components/Icon/Icon.Props.ts

```diff
@@ -64,4 +65,10 @@ export interface IIconProps extends React.HTMLAttributes<HTMLElement> {
    * @memberOf IIconProps
    */
   imageProps?: IImageProps;
+
+  /**
+   * If iconName is undefined or an empty string, a placeholder will be rendered the same size as an icon
+   * @default false
+   */
+  hasPlaceHolder?: boolean;
```

Comment:
Hmmm. Too many props.

Reply:
Could we just check if there is an iconProps that exists? Even if the iconName is blank within it, we could assume that means to leave a placeholder.

The name itself is problematic because INPUT placeholder means something different.
https://www.w3schools.com/TAgs/att_input_placeholder.asp

If we absolutely needed a separate prop I would call it `hasIcon` or something, but really less props results in more consistency and less confusion.

Reply:
I'm happy with iconName="" being enough to render this placeholder icon.